src/ToggleSwitch/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ToggleSwitch'
```

Comment:
I _think_ this should help with the `default` warning. If I remember right, `export *` will export _named_ members but not `default` ones.

```suggestion
export {default} from './ToggleSwitch'
```